skactiveml/visualization/_data_sets.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from scipy.stats import norm, uniform
+from sklearn.utils import check_array
+
+from skactiveml.utils import check_scalar, check_type, check_random_state
+from skactiveml.visualization._misc import _check_interval_and_assign
+
+
+def gaussian_noise_generator_1d(
+    X, *intervals, interval_std=1, default_std=0.5, random_state=None
```

Comment:
```suggestion
    X, intervals, interval_std=1, default_std=0.5, random_state=None
```
Define intervals directly as list similar to centers in sklearn.data_sets.make_blobs.